sdk/keyvault/perf-tests/keyvault-keys/test/cryptography/decrypt.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { RsaEncryptionAlgorithm } from "@azure/keyvault-keys";
+import { randomBytes } from "crypto";
+import { CryptographyTest } from "./cryptography.spec";
+
+export class DecryptTest extends CryptographyTest {
+  public options = {};
+  private cipherText?: Uint8Array;
+  private algorithm: RsaEncryptionAlgorithm = "RSA-OAEP-256";
+
+  async setup(): Promise<void> {
+    const encryptResult = await CryptographyTest.cryptoClient!.encrypt({
+      algorithm: this.algorithm,
+      plaintext: randomBytes(32)
```

Comment:
make it a param?

Reply:
Harsha mind just clarifying what the benefit would be of making this configurable? It can be anything as all we really want is to decrypt it without errors